tasks.py

```diff
@@ -25,7 +26,7 @@ def test(c):
 @task
 def lint(c):
 	run_arg = "pylint -j 4 nut/Config.py tests/ nut/Nsps.py nut/Hex.py nut_gui.py \
-		gui/panes/dirlist.py Fs/driver/http.py"
+		gui/panes/dirlist.py Fs/driver/http.py nut/config_impl/download.py"
```

Comment:
rather than create a config_impl folder, why not create a folder called config, rename config.py to config/__init__.py, then add all of the sub class files like download into that folder?

Reply:
There're 152 occurrences of Config class usage in code. I've decided not to touch it for now and use config_impl folder name for extracted pieces of Config after refactoring

Reply:
> There're 152 occurrences of Config class usage in code. I've decided not to touch it for now and use config_impl folder name for extracted pieces of Config after refactoring

when you create classname/__init__.py, its referenced the same as classname.py.  Doing this change should require no code changes.

Reply:
Not so easy. I've done what you propose:
```
 File "/Users/sergey/Projects/nut/translator/__init__.py", line 13, in <module>
    _locale = Config.region
AttributeError: module 'nut.Config' has no attribute 'region'
```

Reply:
So many updates in different places will be needed unfortunately

Reply:
it should work if you define region in \_\_init\_\_.py